docs/UserGuide.adoc

```diff
@@ -556,6 +610,27 @@ Deletes the command alias with the name `rm`.
 Deletes the command alias with the name `read`.
 // end::alias[]
 
+=== Command autocorrection system _(since v1.5)_
+
+It is common to mispell words when you are typing fast, and when this happens, it is always a hassle to have to correct it and try again.
+With the command autocorrection system, this may only be a press of kbd:[Enter] away.
+
+****
+* Corrects inputed command word which is one letter away from an actual command word, and suggests the correction to you.
+** May not correct for additional space in the command word.
+* You can execute the corrected command by pressing kbd:[Enter].
+****
+
+== Other Features
+
+This section documents features in Bibliotek which are neither book management nor convenience features.
```

Comment:
`This section documents some additional features in Bibliotek that do not fall into the above categories.`?